torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -981,16 +996,23 @@ class TensorExprFuser {
           return false;
         }
       }
-      // cant support non-constant pin_memory or pin_memory = True
```

Comment:
Let's add the autocast nodes here: https://github.com/pytorch/pytorch/blob/95e6722e852d9d13f528945648b5ac6f086983c1/torch/csrc/jit/passes/tensorexpr_fuser.cpp#L622